en-US/Design.xml

```diff
@@ -91,7 +91,7 @@
 				For example, for a button labeled <guibutton>Save As...</guibutton>, do not include the ellipsis in the documentation.
 			</para>
       <para>
-        In most cases, do not include the object type in instructions.
+        In most cases, do not include the GUI element type in instructions.
```

Comment:
Except it might not be a GUI. It might be a TUI, or something else. I think that's why we went with "object" in the first place. You could just use "element type" I guess?